docs/UserGuide.adoc

```diff
@@ -242,35 +242,37 @@ Examples:
 If currently viewing list of contacts, returns contacts (e.g. `john` and `John Doe`).
 If currently viewing list of activities, returns activities (e.g `lunch with john`).
 
-=== Deleting a contact, activity or expense (v1.4) : `delete`
+=== Deleting a contact, activity or expense: `delete`
 
-Deletes the specified index from the current list view. Delete can also be used to delete all entries by not specifying any index, but the user will be prompted to confirm this action.
+Deletes the contact/activity/expense at the specified index from the current display. +
 
-If viewing a contact instead, the contact will be deleted.
+If current display is listing contacts `list c/`, the contact at that specified index will be deleted if he/she is not involved in any activities.
 
-If viewing an activity instead, it will delete an expense* by index. Not specifying any index will result in the current activity being deleted. +
+If current display is listing activities `list a/`, the activity at that specified index will be deleted.
 
-Multiple entries can be deleted at once.
+If viewing an activity instead `view a/1`, it will soft-delete an expense* by index.
```

Comment:
@JohnNzj Will implement the soft-deletion feature tomorrow, hopefully. Presently I've set an expense to strike out it's amount if that expense is soft-deleted. Mind testing if the UI shows such soft-deleted expenses correctly? Thanks!

Reply:
Doesnt work

Reply:
I remember a while ago we intended for soft delete to be such that the very instant anyone was once involved in any expense, they are not removable from the activity. This by extension should mean cannot be deleted. 

Makes life easier for us but likely problematic from a UI/UX standpoint...

Reply:
I think we just prevent them from being deleted once they have been included in any expense, i.e. status quo.